Source/ACE.DatLoader/Entity/BasePropertyDesc.cs

```diff
@@ -36,7 +36,7 @@ public class BasePropertyDesc : IUnpackable
         PropertyInheritanceType m_inheritanceType;
         PropertyDatFileType m_datFileType;
         PropertyPropagationType m_propagationType;
-        PropertyCachingType m_cachingType;
+        //PropertyCachingType m_cachingType;
```

Comment:
@OptimShi This variable was never assigned a value, was that missed in Unpack somewhere?

Reply:
Honestly, so many of these properties/var names were just guessing based on the class details in the PDB that cachingType entry could been any one of those "bool" values, or it could be a run-time value (and I'm sure some of those bool var names are mislabled/flat out wrong, too)